src/plugins/discover/public/application/components/data_grid/data_grid_table.tsx

```diff
@@ -12,9 +12,9 @@ import { DocViewExpandButton } from './data_grid_table_docview_expand_button';
 import { DataGridFlyout } from './data_grid_table_flyout';
 import { DiscoverGridContextProvider } from './data_grid_table_context';
 import { toolbarVisibility } from './constants';
-import { DocViewFilterFn } from '../../doc_views/doc_views_types';
+import { DocViewFilterFn } from '../../../doc_views/doc_views_types';
```

Comment:
My linter is showing that these paths are wrong, can you look at it? Seems to be the case wherever you have updated this path